pages/web-lit-basics.jsx

```diff
@@ -68,6 +68,34 @@ var curriculumList = [
         link: "https://mozilla.github.io/curriculum-final/web-lit-basics-one/session06-story-of-us.html#overview"
       }
     ]
+  },
+  {
+    title: "Privacy Expansion",
+    activities: [
+      {
+        title: "Create Secure Passwords",
+        image1x: "/img/pages/protect-your-data/pass-phrase.jpg",
+        subtitle: "Evaluate, Protect",
+        description: "You will learn several different ways to generate secure passwords and pass-phrases and test their strength, learning web literacy skills like <strong>evaluate</strong> and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session04-create-secure-passwords.html"
```

Comment:
link should be https://mozilla.github.io/curriculum-final/web-lit-basics-one/session07-create-secure-passwords.html#overview ?